### PR TITLE
api: re-sync bpf drop reasons

### PR DESCRIPTION
**Pull request by julianwiedmann**
Looks like we missed adding some bpf DROP_* reasons to the consumers. `DROP_INVALID_TC_BUFFER` being the most recent.

Not sure if these qualify as backport-worthy ...

### CODE REVIEW

File: pkg/monitor/api/drop.go [hunk 1]
```diff
@@ -79,6 +79,7 @@ var errors = map[uint8]string{
 	181: "Policy denied by denylist",
 	182: "VLAN traffic disallowed by VLAN filter",
 	183: "Incorrect VNI from VTEP",
+	184: "Invalid TC buffer",
```
Comment (pchaigno) [inline pkg/monitor/api/drop.go]:
```suggestion
	184: "Failed to update or lookup TC buffer",
```

**Review by pchaigno — APPROVED** [review]

**Comment by pchaigno** [comment]
> Not sure if these qualify as backport-worthy ...

I think it's worth it as the changes are small and they may slightly ease debugging.

**Review by ldelossa — APPROVED** [review]
LGTM

**Comment by joestringer** [comment]
Policy deny makes sense for 1.11, the other two are new in master.

**Review by gandro — APPROVED** [review]
Thank you!

**Review by joamaki — APPROVED** [review]

**Review by nathanjsweet — APPROVED** [review]

**Comment by julianwiedmann** [comment]
/test

rebase so that the CI can pick up new build artifacts

**Comment by gandro** [comment]
GKE is known broken at the moment. Given this PR does not change any datapath behavior and everything else passed, let's mark it ready to merge to hopefully still get it into v1.12.0.

**Pull request merged**